docs/DeveloperGuide.md

```diff
@@ -813,48 +813,116 @@ testers are expected to do more *exploratory* testing.
 
 </box>
 
-### Launch and shutdown
+### Launching and exiting the iVolunteer application
 
 1. Initial launch
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample events and volunteers. The window size may not be optimum.
+   1. Double-click the jar file Expected: Shows the GUI with a set of sample events and volunteers. The window size may not be optimized for your screen.
 
 1. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
+   
+1. Exiting iVolunteer
 
-1. _{ more test cases …​ }_
+   1. Launch the jar file <br> 
+   Expected: Shows the GUI with a set of sample volunteers and events. The window size may not be optimized for your screen.
+   
+   1. Open the help window using the `help` command and minimize it.
+   
+   1. Navigate back to the main window.
 
-### Creating a volunteer
+   1. Open the event show window of the first event using `eshow 1` and minimize it.
 
-1. Prerequisites: NIL.
+   1. Navigate back to the main window and exit from it using the `exit` command. <br> 
+   Expected: The main window and all minimized windows should close automatically.
 
-    1. Test case: `vcreate n/Little Johnny p/98765432 e/littlejohnny@example.com s/little`<br>
-       Expected: Volunteer is created and added to the bottom of the volunteer list. Details of the volunteer shown in the status message.
+### Creating a volunteer
 
-    1. Test case: `vcreate n/Little Johnny p/91234567 e/bigjohnny@example.com s/little`<br>
-       Expected: Volunteer is not created as there already exists a 'Little Johnny' in the volunteer list. Error details shown in the status message.
+1. Test case: `vcreate n/Little Johnny p/98765432 e/littlejohnny@example.com s/little`<br>
+   Expected: Volunteer is created and added to the bottom of the volunteer list. Details of the volunteer shown in the status message.
 
-    1. Other incorrect delete commands to try: `vcreate`, `vcreate n/`, `...` <br>
-       Expected: `Invalid command format!` error message shown in the status message. Instructions on the correct format are shown as well.
+1. Test case: `vcreate n/Little Johnny p/91234567 e/bigjohnny@example.com s/little`<br>
+   Expected: Volunteer is not created as there already exists a 'Little Johnny' in the volunteer list. Error details shown in the status message.
+
+1. Other incorrect delete commands to try: `vcreate`, `vcreate n/`, `...` <br>
```

Comment:
is this supposed to be delete commands or create commands?

Reply:
GOT SOME PARTS I PUT `DELETE` BY ACCIDENT MY BAD I CHANGE IN MY PR

Reply:
idt i did this